db_schema/queries/v1/episodesLatestLTR.sql

```diff
@@ -0,0 +1,56 @@
+WITH
+spotify as (
+  SELECT 
+  spp_date,
+  ep_name as raw_name,
+  `spotifyEpisodePerformance`.`spp_percentile_25` as quarter1, 
+  `spotifyEpisodePerformance`.`spp_percentile_50` as quarter2, 
+  `spotifyEpisodePerformance`.`spp_percentile_75` as quarter3, 
+  `spotifyEpisodePerformance`.`spp_percentile_100` as quarter4,
+  spp_sample_max as listeners
+  FROM spotifyEpisodePerformance
+  LEFT JOIN spotifyEpisodeMetadata USING (episode_id)
+  WHERE
+  spp_date >= @start
+  AND spp_date <= @end
+  AND spotifyEpisodePerformance.account_id = @podcast_id
+),
+apple as (
+  SELECT * 
+  FROM appleEpisodesLTR
+  WHERE
+    date >= @start
+    AND date <= @end
+    AND account_id = @podcast_id 
+),
+-- get the latest date from spotifyEpisodePerformance and appleEpisodeDetails
+latestValidDate as (
```

Comment:
fyi @mre, added this approach to make sure that the date is available aple and spotify . tested it on engkiosk and the whole query was quite fast with 0.02sec